.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
```

Comment:
Files from .idea shouldn't be used in git and loaded to the server.
In order to prevent that and redundant loading of some other files consider creating a .gitignore

Reply:
Ok, I'll do that...later